lhotse/utils.py

```diff
@@ -30,6 +31,47 @@
 _lhotse_uuid: Optional[Callable] = None
 
 
+class SmartOpen:
```

Comment:
Could you copy a part of your described motivation to the description of this class? Also if you could add sth like:

    Example of setting up global credentials::

        >>> SmartOpen.setup(…)

I will re use this doc for Lhotse’s documentation later.